src/components/Admin/index.jsx

```diff
@@ -297,10 +311,21 @@ class Admin extends React.Component {
                 </div>
                 <div className="row mt-4">
                   <div className="col">
-                    <H2 className="table-title">{tableMetadata.title}</H2>
-                    {actionSlug && this.renderResetButton()}
-                    {tableMetadata.subtitle && <H3>{tableMetadata.subtitle}</H3>}
-                    {tableMetadata.description && <p>{tableMetadata.description}</p>}
+                    <div className="row">
+                      <div className="col-12 col-md-3 col-xl-2 mb-2 mb-md-0">
+                        <H2 className="table-title">{tableMetadata.title}</H2>
+                      </div>
+                      <div className="col-12 col-md-9 col-xl-10 mb-2 mb-md-0 mt-0 mt-md-1">
+                        {actionSlug && this.renderUrlResetButton()}
+                        {filtersActive && this.renderFiltersResetButton()}
+                      </div>
+                    </div>
+                    <div className="row">
+                      <div className="col">
+                        {tableMetadata.subtitle && <H3>{tableMetadata.subtitle}</H3>}
```

Comment:
maybe h3 instead of H3?

Reply:
I think, because it worked, that it must be a Bootstrap thing. But I'll look into it.

Reply:
It's not bootstrap, it's specific to this repo. Maybe it has styles attached? Either way, I think it's the right component to use here.

Reply:
ah I was confused then, this is not a regular h tag I see now!!! weird .. but okay.